app/service/utils.py

```diff
@@ -51,3 +53,12 @@ def service_allowed_to_send_to(recipient, service, key_type):
                 whitelist_members
             )
         )
+
+
+def get_current_financial_year_start_year():
```

Comment:
I don't think this is explicitly unit tested. Add unit tests to test_utils?

Reply:
test added.